app/uk/gov/hmrc/claimvatenrolmentfrontend/views/errorPages/enrolment_already_allocated_error_page.scala.html

```diff
@@ -30,12 +31,19 @@
 
     @h1(messages("enrolment_already_allocated.heading"))
 
-    @p {@messages("enrolment_already_allocated.line1")}
+    @p {@messages("enrolment_already_allocated.line_1")}
 
-    @p { @messages("enrolment_already_allocated.line2")
-        <a class='govuk-link' href="appConfig.businessTaxAccountUrl" id="businessTaxAccount" target="_blank">
+    @p { @messages("enrolment_already_allocated.line_2")
+        <a class='govuk-link' href="@appConfig.businessTaxAccountUrl" id="businessTaxAccount" target="_blank">
         @messages("enrolment_already_allocated.link")</a>.
         }
 
+    @govukButton(Button(
+        attributes = Map("id" -> "backToBTA"),
+        content = Text(messages("enrolment_already_allocated.go_back_to_bta")),
+        href = Some(appConfig.btaUrl)
+    ))
+
+    @p {<a class='govuk-link' href="@Routes.SignInOutController.signOut.url" id="signOutLink">@messages("app.common.signOut")</a>}
```

Comment:
There should already be a sign out link at the top right of the page, do we need to show them another one?

Reply:
Anto checked this content yesterday and confirmed he wanted a sign out link after the button. If it's not in a p tag then it appears next to the button rather than below and in the wrong font/design